beachball.config.js

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+/** @type {import("beachball").BeachballConfig } */
+module.exports = {
```

Comment:
It will not, tested by commiting changes to one of the packages we are not publishing, then running `pnpm change`.  This only prompted for entering changes in the packages we publish